### DISCUSSION TITLE
Source or list of available emojis

### DISCUSSION THREAD

**Discussion by epogrebnyak** [thuliteio/doks#361] [Q&A]
Maybe everyone knows but me, but where do the emojis come from that we can use in docs?

Example:

```
{{< alert icon="👉" text="The Tutorial is intended for novice to intermediate users." >}}
```
I found https://gist.github.com/rxaviers/7360908, but not sure if it is a good list.

Emoji is a character, not an image, right?

Can we anyhow reference an emoji `:grey_question:` by name in Hugo/Doks?

**Comment by epogrebnyak** [comment]
https://www.webfx.com/tools/emoji-cheat-sheet/

**Reply by h-enk** [reply]
@epogrebnyak, see: https://gohugo.io/functions/emojify/
